Tapjoy/tpe_prebid_service
# [NGS-1112] Stop sending SChain and PChain to ISX

## JIRA
https://tapjoy.atlassian.net/browse/NGS-1112

## Description
This PR is to stop sending SChain and PChain to ISX as they requested.

## How To Test
https://ui.honeycomb.io/tapjoy/datasets/mediation-bid-service/result/ovpX6H96XwX/trace/GR9BtRVTojB?fields[]=c_name&span=0a81c3b06ff43a73
Please check ISX request to confirm it doesn't have SChain and PChain
